backend/app/routers/person.py

```diff
@@ -13,4 +16,9 @@
 @router.get("/{person_id}")
 async def get_person(person_id: int) -> Person:
     """Specific TV page"""
-    return await get_person_from_id(person_id)
+    if cached_person := (await redis.get(f"person:{person_id}")):
+        return json.loads(cached_person)
+    person = await get_person_from_id(person_id)
+    await redis.set(f"person:{person_id}", person.json())
+    await redis.expire(f"person:{person_id}", 5)

```

Comment:
Maybe the expiration time should be higher? So in future when we can search on people, users get that faster loading :DD

Reply:
Oh, I see what you mean now :DD I thought it was set up 24 hours like the other ones... I was thinking of some other crazy solutions

Reply:
Fixed